load-test-framework/src/main/proto/loadtest.proto

```diff
@@ -16,6 +16,14 @@ service Loadtest {
   rpc Check(CheckRequest) returns (CheckResponse);
 }
 
+service Adapter {
+  // Starts an adapter
+  rpc Start (StartRequest) returns (StartResponse);
+
+  // Executes a command on the adapter, returning the latencies to do so.
```

Comment:
s/to do so/of the operations/

Also, if there is a single command, how come there are multiple latencies (clarify in the comment).